src/encoded/static/components/static-pages/placeholders/MdSortableTable.js

```diff
@@ -0,0 +1,639 @@
+'use strict';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import memoize from 'memoize-one';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['memoize' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4682591)